crates/zkvm/adapters/sp1/src/host.rs

```diff
@@ -66,7 +66,7 @@ impl ZkVmHost for SP1Host {
             std::env::set_var("SP1_PROVER", "mock");
         }
 
-        sp1_sdk::utils::setup_logger();
+        // sp1_sdk::utils::setup_logger();
```

Comment:
NIT: We can’t use this at runtime, correct? Should we remove it, or wrap it inside cfg(test)?

Reply:
Yup makes sense to drop it.